package.json

```diff
@@ -5,11 +5,14 @@
   "main": "build/index.js",
   "scripts": {
     "build": "babel -s -d build examples",
-    "lint": "eslint examples && eslint test",
-    "test": "tape -r babel/register test/index.js",
+    "lint": "eslint examples/ test/ bin/",
+    "test": "tape -r babel-register test/index.js",
     "watch": "watch 'clear && npm run -s lint && npm -s test' test/ examples/",
     "prepublish": "npm run lint && npm run test && npm run build"
   },
+  "bin": {
+    "check-compose": "./bin/check-compose.js"
```

Comment:
What about using `babel-node` so we can write the script in ES6? Haven't tested, but I think the following might work.

```js
"check-compose": ".bin/babel-node some/path/to/check-compose.js"
```